front/sssscs/src/login/Login.tsx

```diff
@@ -1,15 +1,19 @@
-import React, { FormEvent, useState } from "react";
 import Button from '@mui/material/Button';
-import { TextField, Box } from '@mui/material';
+import { TextField, Box, Typography } from '@mui/material';
 import { LoginService, UserLoginDto, UserLoginResultDto } from "./LoginService";
 import { AxiosError, AxiosResponse } from "axios";
+import { FieldValues, useForm } from "react-hook-form";
 
 /**
  * Send a login request to the server.
  * @param data Login credentials.
  */
-const TryLogin = async (data: UserLoginDto) => {
-    LoginService.login(data)
+const TryLogin = async (data: FieldValues) => {
+    const dto: UserLoginDto = {
```

Comment:
Why is this function name capitalized? I thought this would be a component on first read
Usually in React components start with capital letters and hooks start with use[A-Z]

Reply:
https://github.com/magley/sssscs/pull/2/commits/ceec7bb98e542c2181a214165dc4615668b3ce3a